Tasks/plane.js

```diff
@@ -1,17 +1,15 @@
 // Make nested array plane
 
-const_plane = function(arr, res=[]) {
-  j = 0; for(let i = 0, length = arr.length; i < length; i++){
-    value = arr[i];
-    j = i
-    if (Array.isArray(value, typeof value) && [i, length]) {
-    res.push(...const_plane(value));
-    arr[i] = res[i - 1]
-    }else{
-    arr[i] = res[j - 1]
-    res.push(value);
-    };
-  };
+const_plane = array => {
```

Comment:
This looks like an arrow function, maybe you meant 
```suggestion
const plane = array => {
```
Therefore, you'll have to replace 'const_plane' with 'plane' later in the code